docs/horizen_eon/04-develop_and_deploy_smart_contracts/04-local_build_and_deploy.md

```diff
@@ -2,6 +2,7 @@
 title: Local Build and Deploy
 ---
 
+<<<<<<< HEAD
```

Comment:
Delete all the changes in this file.

You can also do `git checkout main docs/horizen_eon/04-develop_and_deploy_smart_contracts/04-local_build_and_deploy.md`